curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646b5c9f34975b060e8e5ae5.md

```diff
@@ -0,0 +1,148 @@
+---
+id: 646b5c9f34975b060e8e5ae5
+title: Step 23
+challengeType: 0
+dashedName: step-23
+---
+
+# --description--
+
+You will target the `select` element and add these styles: 
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed